### ISSUE TITLE
some files could be split into separate ones 

### ISSUE THREAD

**Issue by jackcarlisle** [fcscripters/Devs-In-The-Detail#25]
for example you could have a separate handler.js file and also maybe a separate socket.js file

**Comment by sohilpandya** [comment]
Yes. I would like that as well, but trying that is breaking our whole site. help us!!!! @jackcarlisle
